src/main/scala/com/microsoft/azure/iot/iothub2kafka/IotHubSourceTask.scala

```diff
@@ -55,26 +55,35 @@ class IotHubSourceTask extends SourceTask with LazyLogging with JsonSerializatio
     val topic = props.get(IotHubSourceConfig.KafkaTopic)
     val batchSize = props.get(IotHubSourceConfig.BatchSize).toInt
     val startTime = getStartTime(props.get(IotHubSourceConfig.IotHubStartTime))
+    val eventHubName = props.get(IotHubSourceConfig.EventHubCompatibleName)
+
+    val offsetStorageReader: OffsetStorageReader = if (this.context != null) {
```

Comment:
nitpick: I would use `Option[OffsetStorageReader]`, and `None` instead of `null` -- on line #113 `offsetStorageReader.isDefined`